### ISSUE TITLE
How to mark as spam?

### ISSUE THREAD

**Issue by paulleader** [drcongo/spammy-recruiters#39]
I don't know if I'm just being really thick (entirely possible), but I can't see an option to mark the mail as spam when I create the filter :/

I can delete them, and set the canned response, but not mark as spam.

Obviously filtering them out goes a long was to dealing with my personal problem of spammy recruiters, but it would be nice to actually mark them as spam for the benefit of everyone else :)

**Comment by drcongo** [comment]
I wonder if this feature has been removed from the canned response thing, pretty sure it used to be possible.

**Comment by paulleader** [comment]
Seems like it. I did a bit of googling and the rationale appears to be that
marking things as spam affects all google users, therefore it's not
something they want to encourage.

I can see the logic, bit still a bit annoying.

As a standin I've created a "Recruiter Spam" tag and will just go in there
from time to time and mark them all as spam.

Thanks for the list, running it over old emails pulled out hundreds of
emails, seems pretty accurate :)

On 7 November 2015 at 21:42, Andy Beaumont <notifications@github.com> wrote:

> I wonder if this feature has been removed from the canned response thing,
> pretty sure it used to be possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drcongo/spammy-recruiters/issues/39#issuecomment-154753931>
> .
>



-- 
--
Paul Leader

**Comment by drcongo** [comment]
Seems like a good compromise. Glad you're finding it useful!

**Comment by paulleader** [comment]
Yeah, if I didn't know what a bunch of chancers they all were I'd almost
take it as a complement :/

They've started spamming our entire company (we're small) trying to pimp
people, it's getting so tedious.

On 15 November 2015 at 20:51, Andy Beaumont <notifications@github.com>
wrote:

> Seems like a good compromise. Glad you're finding it useful!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drcongo/spammy-recruiters/issues/39#issuecomment-156852300>
> .
>



-- 
--
Paul Leader
